test/integration/tf_module/README.md

```diff
@@ -0,0 +1,54 @@
+# Forseti Integration Test workflow
+
+This example shows the implementation of an InSpec profile for GCP that depends on the [InSpec GCP Resource Pack](https://github.com/inspec/inspec-gcp).  See the [README](https://github.com/inspec/inspec-gcp) for instructions on setting up appropriate GCP credentials.
+
+
+## Update `attributes.yml.example` to point to your project
```

Comment:
This will be obsolete once this is switched to kitchen-terraform.

Reply:
Updated the documentation in case someone wants to run the inspec tests manually.